diff --git forkSrcPrefix/kernel/model/conf.go forkDstPrefix/kernel/model/conf.go
index 3f7a55f8b..9b5c15658 100644
--- forkSrcPrefix/kernel/model/conf.go
+++ forkDstPrefix/kernel/model/conf.go
@@ -1119,15 +1118,7 @@ func IsSubscriber() bool {
 }

 func IsPaidUser() bool {
-	if IsSubscriber() {
-		return true
-	}
-
-	u := Conf.GetUser()
-	if nil == u {
-		return false
-	}
-	return 1 == u.UserSiYuanOneTimePayStatus
+	return true
 }

 const (
